testrunner.py

```diff
@@ -61,23 +62,7 @@ def setup_test_environment(self, **kwargs):
     def setup_databases(self, **kwargs):
         from corehq.blobs.tests.util import TemporaryFilesystemBlobDB
         self.blob_db = TemporaryFilesystemBlobDB()
-        self.newdbname = self.get_test_db_name(settings.COUCH_DATABASE_NAME)
-        print "overridding the couch settings!"
-        new_db_settings = settingshelper.get_dynamic_db_settings(
-            settings.COUCH_SERVER_ROOT,
-            settings.COUCH_USERNAME,
-            settings.COUCH_PASSWORD,
-            self.newdbname,
-        )
-        settings.COUCH_DATABASE_NAME = self.newdbname
-        for (setting, value) in new_db_settings.items():
-            setattr(settings, setting, value)
-            print "set %s settting to %s" % (setting, value)
-
-        settings.COUCH_SETTINGS_HELPER = settings.COUCH_SETTINGS_HELPER._replace(
-            is_test=True)
-        settings.EXTRA_COUCHDB_DATABASES = settings.COUCH_SETTINGS_HELPER.get_extra_couchdbs()
-
```

Comment:
[settings.py:1082](https://github.com/dimagi/commcare-hq/pull/9809/files#diff-bdf3ecebd8379ca98cc89e545fc90899R1082) - all couch database names are based on `COUCH_DATABASE_NAME`, which now points to a test db when running tests.